### ISSUE TITLE
Alternative, brighter theme

### ISSUE THREAD

**Issue by RoyalBingBong** [odota/ui#811]
Would be great if there was an option to switch to an alternaive theme. For me, bright text on a dark background is rather annoying to read, because my eyes are strained way more and thus after a while, my vision gets a bit blurry and it is harder to focus on the text. This does not happen on sites that have dark text on a bright background.
And according to this older [short blog entry](https://blog.tatham.oddie.com.au/2008/10/13/why-light-text-on-dark-background-is-a-bad-idea/), this is not a rarity!

**Comment by howardchung** [comment]
@blukai @masad-frost opinions?

My main concern with having multiple themes is that it costs additional time due to the need to make sure new things look OK in both themes.

**Comment by masad-frost** [comment]
I agree, I'm usually against 2 different themes, as you said it will slow us down, although should be easier with the color model we're using (using a shared palette). The other problem is branding, the site should be recognizable at a glance, and represent our colors. So I'd rather completely change the design if this is something we're diving into.
